src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const tableBody = document.querySelector('tbody');
+const rowsArray = Array.from(document.querySelectorAll('tbody tr'));
+const headings = document.querySelector('thead');
+const headingsArray = [];
+
+Array.from(headings.firstElementChild.children).forEach(heading => {
+  headingsArray.push(heading.textContent);
+});
+
+function convertToNumber(string) {
+  return parseInt(string.slice(1).replaceAll(',', ''));
+}
+
+headings.addEventListener('click', e => {
+  const index = headingsArray.indexOf(e.target.textContent);
```

Comment:
Instead of finding the index of the clicked header in an array, it would be more efficient to use the cellIndex property of the clicked cell. This would simplify your code and improve performance.